osu.Game/Screens/Edit/Setup/FileChooserLabelledTextBox.cs

```diff
@@ -47,14 +52,16 @@ private void onFileSelected(ValueChangedEvent<FileInfo> file)
 
         public void DisplayFileChooser()
         {
-            Target.Child = new FileSelector(validFileExtensions: ResourcesSection.AudioExtensions)
+            FileSelector fileSelector;
+
+            Target.Child = fileSelector = new FileSelector(validFileExtensions: ResourcesSection.AudioExtensions)
             {
                 RelativeSizeAxes = Axes.X,
                 Height = 400,
-                Anchor = Anchor.Centre,
-                Origin = Anchor.Centre,
                 CurrentFile = { BindTarget = currentFile }
             };
+
+            sectionsContainer?.ScrollTo(fileSelector);
```

Comment:
what's up with this nullcheck? the resolved property does not allow nulls, and surely this can't fire before BDL/injection?

Reply:
￼oh i meant to make it allow null, just in case this component is used in such a way.